src/database/index.tsx

```diff
@@ -192,6 +192,8 @@ export const createBaselineScenario = async () => {
     baselineScenarioRef = await db.collection(scenariosCollectionId).add({
       name: "Baseline Scenario",
       baseline: true,
+      dataSharing: false,
+      dailyReports: false,
```

Comment:
@daschi - We can add a `description` field and default text here. 
@jessex - Do we have copy we can include as a default description?

Reply:
Yep! You can use this:
> Welcome to your new scenario. To get started, add in facility data on the right-hand side of the page. Your initial scenario is also your 'Baseline' - meaning this is where you should keep real-world numbers about the current state of your facilities, their cases, and mitigation steps.

I know this was also used in the fresh start modal but I think that may have been a misunderstanding. I'm going to make sure all copy is where it should be later this weekend, so you can use the above for now.

Reply:
Whoops, accidentally hit Resolve.  Meant to add a comment to let you all know that I added a `description` to the existing Scenarios in firestore with some generic "Default description" copy.

Reply:
Nice! So I think that means we should be unblocked to add description text reads and updates in the sidebar, @daschi.